src/buttonMaker/IconForm.jsx

```diff
@@ -0,0 +1,98 @@
+import * as allIcons from "@chakra-ui/icons";
+import "./buttonMaker.css";
+import { changeForm, clickForm } from "./formLogic";
+
+// A module of all the availabe icons
+let { Icon, createIcon, ...iconsModule } = allIcons;
+
+// A list of all the icon names
+let iconsName = Object.keys(iconsModule).map((e) => e);
+
+// A dropdown for the user to pick their icon
```

Comment:
suggestion: comment would be clearer if it said "dropdown item" rather than just "dropdown"